<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>进程管理器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>提交新任务</string>
    </property>
    <widget class="QLineEdit" name="procName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="priority">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="needTime">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="memory">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>进程名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>优先级</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>所需时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>内存(KB)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>√提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>任务调整</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>进程PID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>阻塞</string>
     </property>
    </widget>
    <widget class="QPushButton" name="awakeButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>唤醒</string>
     </property>
    </widget>
    <widget class="QPushButton" name="killButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>杀死</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>490</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>运行队列</string>
    </property>
    <widget class="QTableView" name="runningTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>470</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>490</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>创建队列</string>
    </property>
    <widget class="QTableView" name="createTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>470</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>490</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>就绪队列</string>
    </property>
    <widget class="QTableView" name="readyTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>470</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>490</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>阻塞队列</string>
    </property>
    <widget class="QTableView" name="blockTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>470</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>121</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>运行任务</string>
    </property>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>运行一个时间</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="messageBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>历史消息</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="systemInfo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
